Dockerfile

```
@@ -1,10 +1,10 @@
-FROM jsii/superchain
+FROM jsii/superchain:1-bookworm-slim-node18
 
 WORKDIR /app
 
 ARG BUILD_ARGS
 
-COPY . .
+COPY --chown=1001:1001 . .
```

Not sure about this change
```suggestion
COPY . .
```